docker/Dockerfile

```diff
@@ -91,8 +91,8 @@ RUN apt-get update && apt-get install -y g++-5-arm-linux-gnueabihf && \
     ln -s /usr/bin/arm-linux-gnueabihf-g++-5 /usr/bin/arm-linux-gnueabihf-g++
 
 # Install requirements
-COPY lmnet/*requirements.txt /tmp/requirements/
-RUN pip install -r /tmp/requirements/gpu.requirements.txt
+COPY ./*requirements.txt /home/blueoil/
```

Comment:
I think we don't need to copy requirement to `/home/blueoil`.
requirements.txt is not required for production use.

Reply:
+1

Reply:
@suttang 
We need `requirements.txt` in Docker image because we need to install requirements.
Which path is good ?

Reply:
@iizukak 
/tmp/requirements/ would be better.